From 42733c56ee1c4c7cfc754319b0fc1080ba7830e0 Mon Sep 17 00:00:00 2001
From: Lin Jianhua <linjh@rock-chips.com>
Date: Thu, 23 Jul 2020 17:12:03 +0800
Subject: [PATCH] ARM: configs: add rv1126-facial-gate.config for rv1109 facial
 gate

make ARCH=arm rv1126_defconfig rv1126-facial-gate.config

Signed-off-by: Lin Jianhua <linjh@rock-chips.com>
Change-Id: I31719cb84af3d5ca12acec44df2976cdaf43e246
---
 arch/arm/configs/rv1126-facial-gate.config | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 arch/arm/configs/rv1126-facial-gate.config

diff --git a/arch/arm/configs/rv1126-facial-gate.config b/arch/arm/configs/rv1126-facial-gate.config
new file mode 100644
index 000000000000..0c08ccde03df
--- /dev/null
+++ b/arch/arm/configs/rv1126-facial-gate.config
@@ -0,0 +1,6 @@
+# CONFIG_VIDEO_GC4C33 is not set
+# CONFIG_VIDEO_IMX347 is not set
+# CONFIG_VIDEO_OS04A10 is not set
+CONFIG_VIDEO_OV2718=y
+# CONFIG_VIDEO_OV4689 is not set
+CONFIG_LEDS_PWM=y
-- 
2.35.3

